bitnami/redis/Chart.yaml

```diff
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/redis
-version: 17.0.10
+version: 18.0.0
```

Comment:
In this case, a major version is not longer needed.

Reply:
:heavy_check_mark: